lib/specs.js

```diff
@@ -478,6 +478,10 @@ Reader.prototype._readEnum = function (schema) {
   do {
     schema.symbols.push(tk.next().val);
   } while (!tk.next({val: '}', silent: true}) && tk.next({val: ','}));
+
+  if (tk.next({val: '=', silent: true})) {
+    schema.default = tk.next().val;
+  }
```

Comment:
It looks like the spec requires a semicolon in the case with a default value, we should add a statement to parse it (`tk.next({val: ';'});`).

Reply:
I added the statement to parse the semicolon when it finds an enum with a default value. Can you point to me in the avro spec where it mentions it? I wasn't able to find it for my reference. Thanks!

Reply:
https://avro.apache.org/docs/1.9.1/idl.html#format_enums